src/main/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionService.java

```diff
@@ -314,26 +348,38 @@ Map<String, String> getPostComputePalaemonTokens(PalaemonSessionRequest request)
         // encryption
         Map<String, String> encryptionTokens = getPostComputeEncryptionTokens(request);
         if (encryptionTokens.isEmpty()) {
-            throw new Exception("Failed to get post-compute encryption tokens - taskId: " + taskId);
+            // This branch should never happen
+            throw new TeeSessionGenerationException(
+                    POST_COMPUTE_GET_ENCRYPTION_TOKENS_FAILED_UNKNOWN_ISSUE,
```

Comment:
I feel we can rename remove `_UNKNOWN_ISSUE` suffix in these

Reply:
I like the idea of explicitly telling something went wrong but we do not exactly know what went wrong. That's where `_UNKNOWN_ISSUE` seems useful. Would you really prefer `POST_COMPUTE_GET_ENCRYPTION_TOKENS_FAILED`?

Reply:
I see ok :)